keyboards/ibm/model_m_4th_gen/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "ibm/model_m_4th_gen", 
+    "url": "https://github.com/purdeaandrei/overnumpad_controller_1xb", 
+    "maintainer": "purdeaandrei", 
+    "width": 22.5, 
+    "height": 6.5, 
```

Comment:
```suggestion
```

Reply:
Also added this PR to remove width and height from the KLE converter: https://github.com/qmk/qmk_api/pull/52